src/neo/Persistence/DataCache.cs

```diff
@@ -95,6 +95,7 @@ public void Add(StorageKey key, StorageItem value)
                     {
                         TrackState.Deleted => TrackState.Changed,
                         TrackState.NotFound => TrackState.Added,
+                        TrackState.Added => TrackState.Added,
```

Comment:
It is incorrect. If the state is `Added`, we should throw.

Reply:
But if someone use add twice?